tests/pytests/test_lro_drivers.py

```diff
@@ -10,14 +10,10154 @@
 from ale import util
 from ale.drivers.lro_drivers import LroLrocPds3LabelNaifSpiceDriver
 from ale.drivers.lro_drivers import LroLrocIsisLabelNaifSpiceDriver
+from ale.drivers.lro_drivers import LroMiniRfIsisLabelNaifSpiceDriver
 from ale.transformation import TimeDependentRotation
 
 from conftest import get_image_label, get_isd, get_image_kernels, convert_kernels, compare_dicts
 
 image_dict = {
-    'M103595705LE': get_isd("lrolroc")
+    'M103595705LE': get_isd("lrolroc"),
+    '03821_16N196_S1': {
```

Comment:
Can you split this into a separate file and use get_isd so that the test file is cleaner.

Reply:
Sure!